### DISCUSSION TITLE
Messenger 點擊網址詐騙達高峰！數百萬用戶FB 帳密遭盜洩個資

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#435] [科技新知]
「國外資安廠商 PIXM 發佈最新研究報告指出，利用FB臉書社群平台與 Messenger即時通訊軟體的散播管道，藉此發起大型的網路釣魚惡意攻擊活動，從去年9月就已開始於社群平台上活躍至現在，其攻擊程度的規模於今年四、五月期間，達到最高峰。」
我去年大一準備段考的時候就因為讀到腦霧點到連結進去被騙過，帳號還會自動傳給好友同樣的連結，我認為這是更讓人困擾的。還會最後帳號拿得回來，這也讓我對網路釣魚的警覺性增高不少。

資料來源：https://www.thenewslens.com/article/168126?_gl=1*1cx7lno*_ga*MTJWNUY1YjJEU1BRaGZueWRLRUN4X1U3bHF0SEhPT1pBa2NyVlk2Szl5SmxMZnJ0UFJpTkxsOVU1T2xOSElGMg..

**Comment by YuHengChen54** [comment]
這種釣魚的病毒真的很煩，我也收過很多

**Comment by lx3211** [comment]
之前就常常收到什麼「這個是你嗎？！....」的訊息
那種不熟的朋友傳來的連結絕對不要亂點

**Comment by xCk27x** [comment]
感覺是Messeger在資訊安全這塊沒做好
如果點個連結就能被盜帳號隱憂真的很大

**Comment by NcuJerryKao** [comment]
我朋友 他現在中大資工的 我被釣了 自動傳網址給他 他以為這是我做的釣魚
結過把帳密輸入進去了

此事件後 我對釣魚 有格外警覺
只要網誌怪怪的 明明是fb登入頁面  網址卻不是https://www.facebook.com/ 
若用隨意的帳密點得進去 也都是有問題的 代表他只是html的表單 而我們案的登入按鈕 便是提交表單的button

**Comment by Ben6uo** [comment]
中大國際事務處的人都有被騙過了

**Reply by gnsJhenJie** [reply]
有卦嗎XDD

**Comment by gnsJhenJie** [comment]
公共Wi-Fi也很常有同樣的問題，像是以前有人架設一個Wi-Fi熱點命名為iTaiwan，然後登入網頁用的跟政府iTaiwan的一樣畫面，連到那個Wi-Fi並輸入帳密的人帳密就洩露了。所以後來政府的iTaiwan就取消登入的功能，只要勾選同意使用條款就可以開始用了。

(而且做假iTaiwan的人很有心機，輸入第一次密碼會顯示密碼錯誤，使用者這時候就會再輸入自己的另一個密碼，這樣就可以得到使用者的兩個密碼了)

**Comment by BingRong1217** [comment]
太可怕了...以後一定要小心

**Comment by awkward-willy** [comment]
臉書上本就存在各式各樣的詐騙與釣魚，而 meta 公司、各國政府也盡可能的去防堵，但仍需仰賴使用者加強自己的資安意識，才能盡可能的減少被騙的事情一再上演。

**Comment by limitx0** [comment]
只能自己做好防備
輸入帳密之前要多留意

**Comment by Yilin20021016** [comment]
這樣的話對很多東西都要小心很多 不知道他會不會裝成其他東西來騙

**Comment by larry40305** [comment]
都沒人傳這個給我，是不是代表我沒朋友

**Comment by jcpan821** [comment]
我上次也差點被騙，太可怕了
